ios/Codeforces Watcher/Application/features/users/UserViewController.swift

```diff
@@ -154,9 +183,9 @@ class UserViewController: UIViewControllerWithCross {
         
         let none = "None".localized
         
-        rankLabel.text = "Rank".localizedFormat(args: user.rank ?? none)
-        nameLabel.text = "Name".localizedFormat(args: user.firstName ?? "", user.lastName ?? none)
-        currentRatingLabel.text = "Current rating".localizedFormat(args: user.rating ?? none)
-        maxRatingLabel.text = "Max rating".localizedFormat(args: user.maxRating ?? none)
+        rankLabel.attributedText = colorTextByUserRank(text: user.rank?.capitalized ?? none, rank: user.rank)
+        handleLabel.attributedText = colorTextByUserRank(text: user.handle, rank: user.rank)
+        ratingLabel.attributedText = ratingViewByUserRank(text: "Rating".localizedFormat(args: user.rating ?? none, user.maxRating ?? none), currentRank: user.rank, maxRank: user.maxRank)
+        contributionLabel.attributedText = contributionView(text: "Contribution".localizedFormat(args: user.contribution ?? none))
```

Comment:
Can be simplified with

```
contributionLabel.attributedText = user.contributionText

extension User {
    
    var contributionText: String {
        if let contribution = user.contribution {
            return contributionView(text: "Contribution".localizedFormat(args: contribution))
        } else {
            return AttributedString(text: "no_contribution".localized)
        }
    }
}
```

Don't hesitate to write more code if it improves readability.